src/components/Card/CardReadMore.js

```diff
@@ -1,28 +1,46 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import classNames from 'classnames'
+import { Route, Link, BrowserRouter as Router } from 'react-router-dom'
 
 import Icon from '../Icon/Icon'
 
+const tagPropType = PropTypes.oneOfType([
+  PropTypes.func,
+  PropTypes.string,
+  PropTypes.shape({ $$typeof: PropTypes.symbol, render: PropTypes.func }),
+  PropTypes.arrayOf(PropTypes.oneOfType([
+    PropTypes.func,
+    PropTypes.string,
+    PropTypes.shape({ $$typeof: PropTypes.symbol, render: PropTypes.func }),
+  ]))
+]);
+
```

Comment:
I don't think we need to define tagPropTypes here. We can leverage that from the `reactstrap` library.